install_julia.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+apt-get -qq update
+DEBIAN_FRONTEND=noninteractive apt-get -qqy install ${CI_APT_INSTALL:-}
```

Comment:
why not fold this into another apt-get invocation that needs to execute anyway, as it used to be? e.g. the initial installation of wget/curl, to avoid useless invocations of apt-get